src/network/mcpe/NetworkSession.php

```diff
@@ -149,7 +149,7 @@ class NetworkSession{
 	private bool $disconnectGuard = false;
 	private bool $loggedIn = false;
 	private bool $authenticated = false;
-	private int $connectTime;
+	public int $connectTime;
```

Comment:
why do you make it externally editable and not just private readonly?

Reply:
Is Symply really simple ? or ?... xd